src/main/java/pl/szczesniak/dominik/whattowatch/users/domain/model/exceptions/UsernameIsTakenException.java

```diff
@@ -0,0 +1,9 @@
+package pl.szczesniak.dominik.whattowatch.users.domain.model.exceptions;
+
+public class UsernameIsTakenException extends RuntimeException {
```

Comment:
Czemu ten wyjątek jest w domenie? Przecież on jest rzucany tylko przez repo. To jest wyjątek z warstwy persystencji (infrastruktury do przechowywania danych), a nie z domeny